src/PetProject.Framework.Kafka/Consumer/IConsumer.cs

```diff
@@ -16,11 +13,6 @@ public interface IConsumer<TBaseMessage> : IDisposable
         /// </summary>
         void StartConsuming();
 
-        /// <summary>
-        /// Decorator around Confluent Consumer to commit messages asynchronously after success.
-        /// </summary>
-        Task<CommittedOffsets> CommitAsync();
```

Comment:
Don't delete this method. It is needed, for example, I will only commit the offset after the command/event has succeeded in its operation. 

Otherwise, you are removing the possibility to commit offsets in the consumer.

Only the retry will commit if the auto.commit property is disabled.

Reply:
Let's keep the discussion about this topic on the comment above https://github.com/the-pet-projects/framework-kafka/pull/13#discussion_r155926618